src/index.ts

```diff
@@ -12,37 +12,72 @@ import 'css/editor.scss';
 import IPPSettings from './PPSettings.interface';
 declare const PP_SETTINGS: IPPSettings;
 
+// Optional external hardcoded annotations
+import input_annotations from '../config/annotations.json';
```

Comment:
do we have a `json-loader` in webpack config or is it handled by one of the currently used ones?

Reply:
Webpack 2+ has a built-in json loader ;)